samples/maps/demo/blend-colors/demo.js

```diff
@@ -35,9 +47,17 @@
                 }
             },
 
+            mapView: {
+                projection: {
+                    name: 'WebMercator'
+                },
+                zoom: 3.35,
+                center: [15, 50.5]
+            },
+
             plotOptions: {
                 mapbubble: {
-                    minSize: 20,
+                    minSize: 30,
                     maxSize: 200,
                     opacity: 0.3,
```

Comment:
I personally like more vivid colors. What about setting opacity to 0.5?

It's interesting how differently we can show the same data by manipulating bubbles sizes 🤔 
FYI @KacperMadej @madepiet @DJTechnoo @TorsteinHonsi  if you have any insights about the design

Or maybe we could add an update button to let users see how the same data can be presented differently depending on different bubbles sizes?

Reply:
Here are some options for the demo that could be discussed - https://jsfiddle.net/BlackLabel/jaugx91p/2/ - changes are in code and we can discuss them on tomorrow's daily.

Reply:
@KacperMadej demo looks great to me!

Reply:
I agree with the vivid colors, 0.4 or 0.5 look a bit fresher.

As for the design in general, I will tinker a bit with that before the final publish. In particular, I think we should drop the spacing/margin, so that the map extends to the edge. Also the title and subtitle are a bit long, which becomes more apparent when emulating mobile phone.